recipes/libtommath/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+context:
+  version: "1.3.0"
+
+package:
+  name: libtommath
+  version: ${{ version }}
+
+source:
+  url: https://github.com/libtom/libtommath/archive/v${{ version }}.tar.gz
+  sha256: 6d099e93ff00fa9b18346f4bcd97dcc48c3e91286f7e16c4ac5515a7171c3149
+
+build:
+  number: 0
+  skip:
+    - osx
```

Comment:
Mac OS and Linux builds should be very similar, so I think you can remove this skip.